tests/expressionEditor.js

```diff
@@ -0,0 +1,39 @@
+/*jshint multistr: true */
+require([
+    'chai',
+    'jquery',
+    'tests/utils',
+    'vellum/javaRosa',
+    'vellum/util',
+    'text!static/javaRosa/text-with-constraint.xml'
+], function (
+    chai,
+    $,
+    util,
+    jr,
+    vellum_util,
+    TEXT_WITH_CONSTRAINT_XML
+) {
+    var assert = chai.assert;
+
+    describe("The javaRosa plugin with multiple languages", function () {
```

Comment:
Either the name or the location of this test are incorrect. Out of curiosity, is it testing anything related to appearance?